assets/styles/_variables.scss

```diff
@@ -2,6 +2,7 @@
 $white: #fff !default;
 $black: #000 !default;
 $red: #e85b55 !default;
+$bright-red: #e32926 !default;
```

Comment:
I've asked designer, you can remove this variable and set `$red: #e32926 !default;`